subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/KotlinPluginSmokeTest.groovy

```diff
@@ -17,31 +17,34 @@
 package org.gradle.smoketests
 
 import org.gradle.integtests.fixtures.android.AndroidHome
+import org.gradle.testkit.runner.BuildResult
 import org.gradle.util.Requires
 import spock.lang.Unroll
 
 import static org.gradle.testkit.runner.TaskOutcome.SUCCESS
 import static org.gradle.util.TestPrecondition.KOTLIN_SCRIPT
 
 class KotlinPluginSmokeTest extends AbstractSmokeTest {
+
     @Unroll
-    def 'kotlin #version plugin'() {
+    def 'kotlin #version plugin, workers=#workers'() {
         given:
         useSample("kotlin-example")
         replaceVariablesInBuildFile(kotlinVersion: version)
 
         when:
-        def result = runner('run').forwardOutput().build()
+        def result = build(workers, 'run')
 
         then:
         result.task(':compileKotlin').outcome == SUCCESS
 
         where:
-        version << TestedVersions.kotlin
+        version << TestedVersions.kotlin * 2
```

Comment:
❓ The multiplications are confusing, what's their intent?

If the goal is to test all permutations, maybe something like the following would work best?
```groovy
where:
[version, workers] << [TestVersions.kotlin, [true, false]].combinations()
```